watertap/unit_models/tests/test_gac.py

```diff
@@ -60,421 +56,604 @@
+        ms.fs.unit.b0.fix(0.784576)
+        ms.fs.unit.b1.fix(0.239663)
+        ms.fs.unit.b2.fix(0.484422)
+        ms.fs.unit.b3.fix(0.003206)
+        ms.fs.unit.b4.fix(0.134987)
+
+        return ms
 
     @pytest.mark.unit
     def test_config_simplified(self, gac_frame_simplified):
-        m = gac_frame_simplified
+        ms = gac_frame_simplified
         # check unit config arguments
-        assert len(m.fs.unit.config) == 8
+        assert len(ms.fs.unit.config) == 9
 
-        assert not m.fs.unit.config.dynamic
-        assert not m.fs.unit.config.has_holdup
-        assert m.fs.unit.config.material_balance_type == MaterialBalanceType.useDefault
+        assert not ms.fs.unit.config.dynamic
+        assert not ms.fs.unit.config.has_holdup
+        assert ms.fs.unit.config.material_balance_type == MaterialBalanceType.useDefault
         assert (
-            m.fs.unit.config.momentum_balance_type == MomentumBalanceType.pressureTotal
+            ms.fs.unit.config.momentum_balance_type == MomentumBalanceType.pressureTotal
         )
         assert (
-            m.fs.unit.config.film_transfer_coefficient_type
+            ms.fs.unit.config.film_transfer_coefficient_type
             == FilmTransferCoefficientType.fixed
         )
         assert (
-            m.fs.unit.config.surface_diffusion_coefficient_type
+            ms.fs.unit.config.surface_diffusion_coefficient_type
             == SurfaceDiffusionCoefficientType.fixed
         )
 
-        assert m.fs.unit.config.property_package is m.fs.properties
-
-        # only designed for single solute and single solvent (water)
-        assert len(m.fs.unit.config.property_package.solute_set) == 1
-        assert len(m.fs.unit.config.property_package.solvent_set) == 1
+        assert ms.fs.unit.config.property_package is ms.fs.properties
+        assert len(ms.fs.unit.config.property_package.solute_set) == 1
+        assert len(ms.fs.unit.config.property_package.solvent_set) == 1
 
     @pytest.mark.unit
     def test_build_simplified(self, gac_frame_simplified):
-        m = gac_frame_simplified
+        ms = gac_frame_simplified
 
         # test ports
         port_lst = ["inlet", "outlet", "adsorbed"]
         for port_str in port_lst:
-            port = getattr(m.fs.unit, port_str)
-            assert (
-                len(port.vars) == 3
-            )  # number of state variables for NaCl property package
+            port = getattr(ms.fs.unit, port_str)
+            assert len(port.vars) == 3  # number of state variables for property package
             assert isinstance(port, Port)
 
         # test statistics
-        assert number_variables(m) == 77
-        assert number_total_constraints(m) == 45
-        assert number_unused_variables(m) == 10  # dens parameters from properties
+        assert number_variables(ms) == 77
+        assert number_total_constraints(ms) == 44
+        assert number_unused_variables(ms) == 10  # dens parameters from properties
 
```

Comment:
I noticed this is missing in some of our other unit model tests, but I think you should add an `assert_units_consistent(ms)` here.